src/reformat.c

```diff
@@ -109,7 +109,10 @@ avifBool avifPrepareReformatState(const avifImage * image, const avifRGBImage *
     state->rgbMaxChannel = (1 << rgb->depth) - 1;
     state->yuvMaxChannelF = (float)state->yuvMaxChannel;
     state->rgbMaxChannelF = (float)state->rgbMaxChannel;
-    state->uvBias = 1 << (image->depth - 1);
+    state->yBias = (float)(state->yuvRange == AVIF_RANGE_LIMITED ? (16 << (state->yuvDepth - 8)) : 0);
+    state->uvBias = (float)(1 << (state->yuvDepth - 1));
+    state->yRange = state->yuvRange == AVIF_RANGE_LIMITED ? ((float)(235 << (state->yuvDepth - 8)) - state->yBias) : state->yuvMaxChannelF;
+    state->uvRange = state->yuvRange == AVIF_RANGE_LIMITED ? ((float)(240 << (state->yuvDepth - 8)) - state->yBias) : state->yuvMaxChannelF;
```

Comment:
Lines 114-115 are equivalent to:

```
    state->yRange = state->yuvRange == AVIF_RANGE_LIMITED ? (float)(219 << (state->yuvDepth - 8)) : state->yuvMaxChannelF;
    state->uvRange = state->yuvRange == AVIF_RANGE_LIMITED ? (float)(224 << (state->yuvDepth - 8)) : state->yuvMaxChannelF;
```

This form is more similar to Equations 23-25 in H.273.